composer.json

```diff
@@ -9,7 +9,7 @@
     },
     "minimum-stability": "stable",
     "require": {
-        "php": ">=5.4,<8.0-dev",
+        "php": "~5.6.0",
```

Comment:
Hmhmh kinda iffy, with `ignore-platform-reqs` we could potentially install PHP 5-only dependencies. With all these considerations we have to make, I'm leaning more and more to just going back to requiring PHP ~5.6|~7.0 so we can reliably test PHP 7 as well, or dropping testing of PHP 7 entirely. Also, shouldn't the tests prove (to a certain degree) that the application works?